---
layout: cv
title: Hugo Tavares CV
---

# hugo tavares
Bioinformatician & Data Analyst

Sainsbury Laboratory Cambridge University

<div id="webaddress">
  <a href=""><i class="fas fa-at"></i> hugo.tavares @ slcu.cam.ac.uk</a> |
  <!-- <a href="https://tavareshugo.github.io"><i class="fas fa-home"></i> tavareshugo.github.io</a><br/> -->
  <a href="https://github.com/tavareshugo"><i class="fab fa-github"></i> tavareshugo</a> |
  <a href="https://orcid.org/0000-0001-9373-2726"><i class="ai ai-orcid"></i> 0000-0001-9373-2726</a>
</div>


## Education

`2010-2014`
**PhD in Evolutionary Genetics and Genomics**, John Innes Centre, Norwich, UK

`2006-2007`
**Masters in Evolutionary and Developmental Biology**, University of Lisbon, Lisbon, Portugal

`2002-2006`
**Undergraduate in Biology**, University of Lisbon, Lisbon, Portugal


## Work Experience

* <font color="gray"><i class="fas fa-desktop"></i> = computational work (data analysis, bioinformatics) </font>
* <font color="gray"><i class="fas fa-flask"></i> = experimental work (molecular genetics) </font>

`2015-now`
<i class="fas fa-desktop"></i> **Research Associate**, Leyser lab, SLCU, Cambridge, UK

Analysis of quantitative genetic data for environmentally-sensitive traits in Arabidopsis.
Combining standard association mapping approaches
(GWAS and QTL mapping using multi-parent RILs) with genome scans for selection signals
in artifically selected populations (using an "evolve-and-resequence" approach).

`2014-2015`
<i class="fas fa-desktop"></i> **Research Associate**, Clark lab, The Genome Analysis Centre, Norwich, UK

Draft genome assembly for an Asian honeybee species and genome-wide association mapping
for parasite resistance in European honeybee.

`2010-2014`
<i class="fas fa-desktop"></i> <i class="fas fa-flask"></i> **PhD student**, Coen lab, John Innes Centre, Norwich, UK

Used a combination of molecular genetics, genomics and population genetics to show the evolutionary forces
that shape differentiation patterns across genomes in natural populations.

`2008-2009`
<i class="fas fa-flask"></i> **Research technician**, Almeida lab, ITQB, Oeiras, Portugal

Characterised plant self-incompatibility genes, which are hypervariable genes comprising high
numbers of alleles in populations.

`2006-2007`
<i class="fas fa-flask"></i> **Masters thesis**, Almeida lab, ITQB, Oeiras, Portugal

Molecular isolation of transposons to be used for mutagenesis in non-model species.


## Bioinformatics Support

` `
**Data analysis**: initiated and run weekly "drop-in" sessions for SLCU researchers to get advice on data analysis and
regularly support and collaborate with colleagues in the Leyser group

` `
**HPC cluster**: co-maintainer of the SLCU cluster: install software, create accounts and support new
users

` `
**Community building:** organise data analysis activities at SLCU to foster ideas exchange: bi-weekly "statistics lunch chats" (currently running), a book club (finished), bioinformatics club (finished)

` `
**Data management:** member of the _Data Champions_ initiative at Cambridge University, whose purpose is to support researchers with research data management, sharing and reproducible data analysis

` `
**Computing support**: member of the _SLCU Computing Committee_, participating in regular meetings to discuss computational
needs in the institute


## Training attented

`Mar 2018`
**Computer vision with R and Python** and **Image Recognition & Deep Learning**, Leuven University, Belgium

`Oct 2018`
**Introduction to Bayesian statistics using BUGS**, University of Cambridge, UK

`Nov 2016`
**Data Carpentry instructor training**, University of Cambridge, UK

`Jul 2016`
**ELIXIR Train the Trainer**, University of Cambridge, UK

`Sep 2015`
**R object-oriented programming and package development**, University of Cambridge, UK


## Teaching and Supervision

### Masters students

**Gregory Tate**, Part III Systems Biology thesis, University of Cambridge, UK

**Elisabeth Forsyth**, Part III Systems Biology thesis, University of Cambridge, UK

### Undergraduate students

**Liron Shenhav**, Bachelors thesis, Van Hall Larenstein, University of Applied Sciences, Netherlands

### Data analysis training

Certified **_[Data Carpentry](http://datacarpentry.org/)_ instructor** and teaching material contributor

`2016-now`
Regular **R instuctor** at the _Cambridge University Bioinformatics Training_ facility: 100+ hours of teaching across 18 courses

`2018 & 2019`
3-day course on **Exploratory Data Analysis: Data Carpentry in R** at _EMBL Heidelbergh_
(development of new lesson material)

`2018`
Co-organised a modular **Introduction to R** course tailored for _SLCU_ researchers
(development of new lesson material)

`2018`
2-day **Data Carpentry** course at _Kings College London_ (development of new lesson material)

`2017`
2-day **Data Carpentry** course at the _Francis Crick Institute_, London


## Skills

### Programming

<i class="fas fa-terminal"></i> Shell scripting |
<i class="fab fa-r-project"></i> R programming |
<i class="fab fa-python"></i> Python programming |
<i class="fas fa-network-wired"></i> High performance computing cluster

### Bioinformatics

Analysis of whole-genome Illumina data, including
a wide range of bioinformatic tools with applications in transcriptome analysis,
genetic mapping-by-sequencing, association mapping, population genomics, variant
calling and de-novo genome assembly.


### Data analysis

Expert in R statistical programming language, including packages for data manipulation
and visualisation (e.g. data.table, dplyr, ggplot2), statistical modelling (e.g. lme4)
and BioConductor packages (e.g. DEseq2, GenomicAlignments)


## Publications

### Journals

` `
**Tavares H**, Whibley A, Field DL, ..., Barton NH & Coen E (2018)
_Selection and gene flow shape genomic islands that control floral guides_,
PNAS 115:11006; DOI: [10.1073/pnas.1801832115](https://doi.org/10.1073/pnas.1801832115)

` `
Bradley D, ..., **Tavares H**, ..., Dalmay T & Coen S (2017)
_Evolution of flower color pattern through selection on regulatory small RNAs_,
Science 358:925; DOI: [10.1126/science.aao3526](https://doi.org/10.1126/science.aao3526)


<!-- ### Footer

Last updated: Feb 2019 -->
